content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -80,7 +80,7 @@ different Kubernetes components.
 | `DynamicAuditing` | `false` | Alpha | 1.13 | |
 | `DynamicKubeletConfig` | `false` | Alpha | 1.4 | 1.10 |
 | `DynamicKubeletConfig` | `true` | Beta | 1.11 | |
-| `EndpointSlice` | `false` | Alpha | 1.16 | |
+| `EndpointSlice` | `true` | Beta | 1.17 | |
```

Comment:
instead of changing line 83 directly, you are expected to do something like this:
```
| `EndpointSlice` | `false` | Alpha | 1.16 | 1.16 |
| `EndpointSlice` | `true` | Beta | 1.17 | |
```
The point is that we will still let people know the feature is available as Alpha for 1.16, but only for 1.16.

Reply:
Fixed, thanks!